lib/run.js

```diff
@@ -220,8 +220,10 @@ module.exports = function (options, callback) {
 
             // override the `done` event to fire the end callback
             callbacks.done = function (err) { // @todo - do some memory cleanup here?
+                var collectionDetails = _.pick(options.collection, ['id', 'name']);
+
                 emitter.emit('done', err, emitter.summary); // we now trigger actual done event which we had overridden
-                callback(err, emitter.summary);
+                callback(err, _.merge(emitter.summary, collectionDetails));
```

Comment:
With this methodology, only the end consumer of the summary object will have access to these details.
Ideally, it would be better to pass these details as options to the `RunSummary` constructor, so that the summary always has them, and everyone can use them.

https://github.com/postmanlabs/newman/pull/471/files#diff-de78b50723ff596d7bd35665f36a6cbfR166